### ISSUE TITLE
About the source code of model training？

### ISSUE THREAD

**Issue by JiaWeiGZU** [Pandede/WPODNet-Pytorch#1]
Can you kindly send  the source code of model training and the loss function？
